fhd_core/HEALPix/healpix_cnv_generate.pro

```diff
@@ -151,7 +163,9 @@ FOR i=0L,n_img_use-1L DO BEGIN
     *ija[i]=ija0
         
 ENDFOR
-
+;sa=weighting function?
```

Comment:
`ija` and `sa` are written in the sparse matrix notation of `sprsax2.pro`, in FH/fhd_core/gridding . The `ija`s are the coordinates with data in the sparse matrix, and the `sa`s are the values.